src/covid_model_seiir_pipeline/paths.py

```diff
@@ -44,7 +44,8 @@ def make_dirs(self):
 class ScenarioPaths(Paths):
```

Comment:
Changes to this file pulled in from https://github.com/ihmeuw/covid-model-seiir-pipeline/pull/68